Postinstall issues

Hi,

I'm using `simple-icons-font` in one of projects.

Unfortunately, after update to 4.6.0 and npm install, each time I'm getting:
```
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! simple-icons-font@4.6.0 postinstall: `is-ci || husky install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the simple-icons-font@4.6.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\tomek\AppData\Roaming\npm-cache\_logs\2021-01-16T02_55_59_755Z-debug.log
```

This wasn't issue for 4.5.0, but after upgrade it started. I'm using postinstall in my project.

--- a/package.json
+++ b/package.json
@@ -25,9 +25,9 @@
     "clean": "rm -rf font/ preview/testpage.html screenshot.png",
     "format": "prettier --write --single-quote .",
     "lint": "prettier --check --single-quote .",
-    "prepare": "is-ci || husky install",
-    "prepublishOnly": "npm run build",
-    "postpublish": "npm run clean",
+    "prepublishOnly": "pinst --disable && npm run build",
+    "postinstall": "is-ci || husky install",
+    "postpublish": "pinst --enable && npm run clean",
     "test": "npm run build:testpage && anywhere -h localhost -d . -f /preview/testpage.html"
   },
   "devDependencies": {
@@ -36,6 +36,7 @@
     "clean-css": "4.2.3",
     "husky": "5.0.6",
     "is-ci": "2.0.0",
+    "pinst": "2.1.1",
     "prettier": "2.2.1",
     "pug": "3.0.0",
     "punycode": "2.1.1",

--- a/package.json
+++ b/package.json
@@ -25,8 +25,8 @@
     "clean": "rm -rf font/ preview/testpage.html screenshot.png",
     "format": "prettier --write --single-quote .",
     "lint": "prettier --check --single-quote .",
+    "prepare": "is-ci || husky install",
     "prepublishOnly": "npm run build",
-    "postinstall": "is-ci || husky install",
     "postpublish": "npm run clean",
     "test": "npm run build:testpage && anywhere -h localhost -d . -f /preview/testpage.html"
   },

